src/app/destinyTrackerApi/d2-reviewsFetcher.js

```diff
@@ -20,23 +20,25 @@ class D2ReviewsFetcher {
     this._perkRater = new D2PerkRater();
   }
 
-  _getItemReviewsCall(item) {
+  _getItemReviewsCall(item, platformSelection) {
+    const queryString = `reviews?page=1&platform=${platformSelection}`;
+
     return {
       method: 'POST',
-      url: 'https://db-api.destinytracker.com/api/external/reviews?page=1', // TODO: pagination
+      url: `https://db-api.destinytracker.com/api/external/reviews?${queryString}`, // TODO: pagination
```

Comment:
`reviews?` in both the url and the queryString

Reply:
Thanks for noticing!